### DISCUSSION TITLE
How to self-host Riju

### DISCUSSION THREAD

**Discussion by pr13260** [raxod502/riju#149] [Q&A] [answered]
Kindly answer it 💙

**Comment by raxod502** [comment, marked as answer]
This is documented at https://github.com/raxod502/riju/blob/master/doc/selfhosting.md; however, please be aware that some manual work may be required on your end because the project is still in its early stages and the experience of setting up a new installation from scratch is unnecessarily difficult.

**Reply by pr13260** [reply]
Thanks a lot 😀😀
